File: API/ContaCentral/ContaCentral/Domain/Models/Deposito.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ContaCentral.Domain.Models
{
    public class Deposito
    {
        public int Id { get; set; }
        public decimal Valor { get; set; }
        public DateTime DataHora { get; set; }
        public string Descricao { get; set; }
        public string externalName { get; set; }
        public int externalId { get; set; }

        public int CarteiraId { get; set; }
        [ForeignKey("CarteiraId")]
        public Carteira Carteira { get; set; }
    }
}

File: Website/src/app/domain/models/deposito.ts
export class Deposito {

	id : number = 0
	dataHora: string = ""
	valor: number = 0
	descricao: string = ""
	externalName: string = ""
	externalId: string = ""
	carteiraId: string = ""

	constructor(id: number, dataHora: string, valor: number, descricao: string, externalId: string, externalName: string, carteiraId: string) {
		//specify your own constructor logic
		this.id = id;
		this.dataHora = dataHora;
		this.valor = valor;
		this.descricao = descricao;
		this.externalName = externalName;
		this.externalId = externalId;
		this.carteiraId = carteiraId;
	}
	
}